modules/apps/change-tracking/change-tracking-test/src/testIntegration/java/com/liferay/change/tracking/internal/closure/test/CTClosureFactoryImplTest.java

```diff
@@ -194,8 +194,7 @@ public void tearDown() throws Exception {
 					"CTCollectionLocalServiceImpl",
 				LoggerTestUtil.WARN)) {
 
-			_ctCollectionLocalService.deleteCTCollection(
-				_ctCollection.getCtCollectionId());
+			_ctCollectionLocalService.deleteCTCollection(_ctCollection);
```

Comment:
@jorgediaz-lr Based on the sub tasks in LPS-145936, I think it's better to send this commit in a separate sub task, so it can be merged quickly.